src/platform_impl/ios/view.rs

```diff
@@ -238,6 +279,9 @@ declare_class!(
     }
 );
 
+unsafe impl NSObjectProtocol for WinitView {}
+unsafe impl UIGestureRecognizerDelegate for WinitView {}
```

Comment:
Implement these inside of `declare_class!`, and move the implementation for `gestureRecognizer:shouldRecognizeSimultaneouslyWithGestureRecognizer:` inside of the impl for `UIGestureRecognizerDelegate`.